src/preact/component/3p-frame.js

```diff
@@ -131,7 +133,7 @@ function ProxyIframeEmbedWithRef(
       name: JSON.stringify(
         dict({
           'host': parseUrlDeprecated(src).hostname,
-          'bootstrap': bootstrap ?? getBootstrapUrl(type),
+          'bootstrap': getBootstrapUrl(type),
```

Comment:
Is `bootstrap` no longer optional?

Reply:
I removed it as a prop because it hasn't been used by other components and I'd like to hear from specific use cases that would like for it to be overridable, so figured it might be better as an opportunity for a feature request than supported by default for a hypothetical-as-far-as-I-know scenario.